src/opentime/rationalTime.cpp

```diff
@@ -125,10 +125,10 @@ RationalTime::from_timecode(std::string const& timecode, double rate, ErrorStatu
 
     int dropframes = 0;
     if (rate_is_dropframe) {
-        if ((rate == 29.97) or (rate == 30000/1001.0)) {
+        if ((rate == 29.97) || (rate == 30000/1001.0)) {
             dropframes = 2;
         }
-        else if ((rate == 59.94) or (rate == 60000/1001.0)) {
+        else if ((rate == 59.94) || (rate == 60000/1001.0)) {
```

Comment:
Does windows not support the `or` and `and` keywords, only `||` and `&&` operators?  I'd prefer we use the `or` and `and` form if compatible.

Reply:
They are not defined in Visual Studio 2019, no idea why. I hadn't seem them before and thought they were GCC/clang specific, but according to cppreference they are standard:
https://en.cppreference.com/w/cpp/language/operator_alternative

This Microsoft doc shows they are missing:
https://docs.microsoft.com/en-us/cpp/cpp/keywords-cpp?view=vs-2019

I guess we could define them as macros on Windows?